_automating-configurations/workflow-templates.md

```diff
@@ -6,9 +6,11 @@ nav_order: 25
 
 # Workflow templates
 
-OpenSearch provides several workflow templates for some common machine learning (ML) use cases, such as semantic or conversational search. 
+OpenSearch provides several workflow templates for some common machine learning (ML) use cases. Using a template simplifies complex setups required for use cases like semantic or conversational search. 
 
-You can specify a workflow template when you call the [Create Workflow API]({{site.url}}{{site.baseurl}}/automating-configurations/api/create-workflow/). To provision the workflow, specify `provision=true` as a query parameter. For example, you can configure [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/) by providing the `local_neural_sparse_search_bi_encoder` query parameter as `use_case`, as shown in the following request:
+For example, you can configure the setup required for [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/) by using the `local_neural_sparse_search_bi_encoder` predefined template. When you use a template, the multi-step setup that required running separate API requests is accomplished with a single request.
```

Comment:
I am also still unsure if we should have this as the first example as the other one simplifies a lot more complexity vs this one that simplifies 3 API calls only